cmd/jaeger/internal/integration/grpc_test.go

```diff
@@ -5,12 +5,47 @@ package integration
 
 import (
 	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/jaegertracing/jaeger/plugin/storage/integration"
 )
 
+type GRPCStorageIntegration struct {
+	E2EStorageIntegration
+
+	server *integration.GRPCServer
+}
+
+func (s *GRPCStorageIntegration) initialize(t *testing.T) {
+	var err error
+	s.server, err = integration.NewGRPCServer()
+	require.NoError(t, err)
+	require.NoError(t, s.server.Start())
+
+	s.Refresh = func(_ *testing.T) {}
+	s.CleanUp = s.cleanUp
+}
+
+func (s *GRPCStorageIntegration) cleanUp(t *testing.T) {
+	require.NoError(t, s.server.Close())
+	s.initialize(t)
+}
+
 func TestGRPCStorage(t *testing.T) {
-	s := &StorageIntegration{
-		Name:       "grpc",
-		ConfigFile: "fixtures/grpc_config.yaml",
+	integration.SkipUnlessEnv(t, "grpc")
+
+	server, err := integration.NewGRPCServer()
```

Comment:
you are already calling initialize() in L45

Reply:
You're right, overlooked it